package.json

```diff
@@ -103,6 +103,7 @@
   "devDependencies": {
     "@types/node": ">=16",
     "autocannon": "^4.5.2",
+    "aws-sdk": "^2.1446.0",
```

Comment:
Any way to not need this? Can't we use what gets installed in the `versions` folder somehow? If we start depending on all our integrations this will become way too bloated.